pyp2spec/license_processor.py

```diff
@@ -0,0 +1,146 @@
+from importlib.resources import open_text
+import json
+import re
+import sys
+
+import click
+import requests
+
+
+class TroveConverter:
+    """Takes care of package license classifiers conversion to the Fedora SPDX format.
+    """
+
+    def __init__(self, classifiers):
+        self.classifiers = classifiers
+        self.spdx_identifiers = self.convert_to_spdx()
+
+    def load_trove2fedora_map(self):
+        """Load the file with Python trove classifiers mapped to SPDX identifiers."""
+
+        with open_text("pyp2spec", "classifiers_to_fedora.json") as map_data:
+            return json.load(map_data)
```

Comment:
I don't know how many times a TroveConverter class is actually used when the app runs, but would it make sense to cache the content of this file globally instead of reading it from disk every time this class is used?